tic_tac_toe/game.py

```diff
@@ -24,18 +28,31 @@ def _position_is_valid(position):
 
     Returns True if given position is valid, False otherwise.
     """
-    pass
-
+    if isinstance(position,int):
```

Comment:
Hey @jsymons thanks for taking the time to look through my code.  Yeah, I'm not really sure what I was thinking about the first error catch.  In a flurry, I was probably thinking about the next problem I had to solve.  

It turns out that the last two conditions do a pretty good job at catching strings.  

if position[0] > 2 or position[1] < 0:
        return False
    if position[1] > 2 or position[1] < 0:
        return False



>>> x = ("hi")
>>> isinstance(x, int)
False
>>> len(x)
2
>>> x[0] > 2
True
>>> x[0] < 0
False
>>> x[1] > 2
True
>>> x[1] < 0
False
>>> 

I guess I just got lucky.   Perhaps the tests should be written to be more picky?  I just wrote it quick and saw it passed tests then moved on.